src/components/ScreenTemplate.tsx

```diff
@@ -62,6 +67,13 @@ export class ScreenTemplate extends React.PureComponent<Props> {
           contentContainerStyle={[styles.contentContainer, contentContainer]}
           refreshControl={refreshControl}
           keyboardShouldPersistTaps={keyboardShouldPersistTaps}
+          onScroll={({ nativeEvent }) => {
+            if (isCloseToBottom || allowedUserClick) {
+              //@ts-ignore
```

Comment:
what is problem here?

Reply:
![Zrzut ekranu 2020-11-16 o 08 49 56](https://user-images.githubusercontent.com/67273035/99225671-b8210600-27e8-11eb-9c0d-19332e808726.png)
that could help

Reply:
can be. done

Reply:
I looked at it again and better would be 
 ({ nativeEvent} )=> {
if (isCloseToBottom !== undefined && allowedUserClick !== undefined) {
isCloseToBottom(nativeEvent) && allowedUserClick()
}
}

Reply:
done